### DISCUSSION TITLE
External values issues from gitlab url

### DISCUSSION THREAD

**Discussion by maxdev42** [argoproj/argo-cd#11256] [Q&A]
Hello,

I am currently having trouble deploying my applications with helm on argocd.

I use an Application ressource and I will go to an ApplicationSet next, that I would copy to you in which I must call on values.yml from another repository in my gitlab.

I try to put the link of the repo directly but it does not work.

I haven't found any other solutions to use values ​​files from another gitlab repository.

Can you help me ?

Thanks in advance !

My code :

My Application ressource file :

`apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: react-docker-app
  namespace: argocd
spec:
  syncPolicy:
    automated:
      selfHeal: true
  project: default
  destination:
    server: https://kubernetes.default.svc
    namespace: argocd
  source:
    repoURL: https://gitlab.com/api/v4/projects/40489526/packages/helm/stable
    targetRevision: 0.8.0
    chart: react-chart
    helm:
      valueFiles:
      - https://gitlab.com/maxdev42-gitops-projects/reactdockerapp2/-/blob/master/deployment/valtues.yaml`

My values.yml from an another repository :

`image:
  repository: registry.gitlab.com/maxdev42/react-docker-app
  tag: "appv8"`

**Comment by kostis-codefresh** [comment]
Have you seen this issue? https://github.com/argoproj/argo-cd/issues/2789
